public/app/features/alerting/unified/rule-list/components/EvaluationGroupWithRules.tsx

```diff
@@ -50,21 +50,6 @@ export const EvaluationGroupWithRules = ({ group, rulesSource }: EvaluationGroup
           );
         }
 
-        if (isRecordingRulerRule(rulerRule)) {
```

Comment:
with this change we are not going to identify recording rule from alerting rule? or maybe i'm missing something?

Reply:
This component (which was used for the v2 POC) is being removed in a later PR :)